code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -0,0 +1,46 @@
+/datum/team/ashwalkers
+	name = "Ashwalkers"
+	var/obj/structure/tendril
+	show_roundend_report = FALSE
+
+/datum/team/ashwalkers/New(starting_members, var/obj/structure/goal)
```

Comment:
Don't modify team new please, it will still work but makes standardized arguments for antag/team panels a pain.